publications-2019.bib

```diff
@@ -1,5 +1,222 @@
+  journal = {Advanced Theory and Simulations}
+}
+
+@article{Freddi2019,
+  doi = {10.1016/j.mechrescom.2019.01.009},
+  url = {https://doi.org/10.1016/j.mechrescom.2019.01.009},
+  year = {2019},
+  month = mar,
+  publisher = {Elsevier {BV}},
+  volume = {96},
+  pages = {29--36},
+  author = {Francesco Freddi},
+  title = {Fracture energy in phase field models},
+  journal = {Mechanics Research Communications}
+}
+
+@article{Bzowski2019,
+  doi = {10.1088/1757-899x/627/1/012018},
+  url = {https://doi.org/10.1088/1757-899x/627/1/012018},
+  year = {2019},
+  month = oct,
+  publisher = {{IOP} Publishing},
+  volume = {627},
+  pages = {012018},
+  author = {Krzysztof Bzowski and Lukasz Rauch and Maciej Pietrzyk},
+  title = {Selection of the optimal heat treatment conditions using advanced phase transformation models},
+  journal = {{IOP} Conference Series: Materials Science and Engineering}
+}
+
+@article{https://doi.org/10.11588/heidok.00027466,
+  doi = {10.11588/HEIDOK.00027466},
+  url = {http://archiv.ub.uni-heidelberg.de/volltextserver/id/eprint/27466},
+  author = {Ponce Bobadilla,  Ana Victoria},
+  keywords = {500 Natural sciences and mathematics,  510 Mathematics},
+  title = {Mathematical Models of Cell Migration and Proliferation in Scratch Assays},
+  publisher = {Heidelberg University Library},
+  year = {2019}
+}
+
+@article{Basak2019,
+  doi = {10.1016/j.cma.2018.08.006},
+  url = {https://doi.org/10.1016/j.cma.2018.08.006},
+  year = {2019},
+  month = jan,
+  publisher = {Elsevier {BV}},
+  volume = {343},
+  pages = {368--406},
+  author = {Anup Basak and Valery I. Levitas},
+  title = {Finite element procedure and simulations for a multiphase phase field approach to martensitic phase transformations at large strains and with interfacial stresses},
+  journal = {Computer Methods in Applied Mechanics and Engineering}
+}
+
+@misc{https://doi.org/10.25673/13842,
+  doi = {10.25673/13842},
+  url = {https://opendata.uni-halle.de//handle/1981185920/13968},
+  author = {Balicki,  Linus},
+  keywords = {Mathematik,  510},
+  language = {de},
+  title = {Eine abstrakte Implementierung der Low-Rank ADI Iteration f\"{u}r Lyapunovgleichungen in pyMOR},
+  publisher = {Otto von Guericke University Library,  Magdeburg,  Germany},
+  year = {2019},
+  copyright = {Creative Commons Attribution Share Alike 4.0 International}
+}
+
+@article{Arbogast2019,
+  doi = {10.1007/s10596-019-09871-2},
+  url = {https://doi.org/10.1007/s10596-019-09871-2},
+  year = {2019},
+  month = aug,
+  publisher = {Springer Science and Business Media {LLC}},
+  volume = {23},
+  number = {5},
+  pages = {1141--1160},
+  author = {Todd Arbogast and Zhen Tao},
+  title = {A direct mixed{\textendash}enriched Galerkin method on quadrilaterals for two-phase Darcy flow},
+  journal = {Computational Geosciences}
+}
+
+@article{https://doi.org/10.13140/rg.2.2.25335.78247,
```

Comment:
This is a thesis.

Reply:
Dissertation of the McGill university are supposed to show up here: https://escholarship.mcgill.ca/
But I cannot find this particular one listed there.

Reply:
A thesis submitted to McGill University in partial fulfillment of the
requirements of the Undergraduate Honours Program

Reply:
So, is it a master's thesis then? I'm confused by that phrasing.

Reply:
https://github.com/dealii/publication-list#commonly-used-entry-templates

According to our readme, this should be flagged as `@MastersThesis` with `note = {Honour's Thesis}` if I understood correctly.